### DISCUSSION TITLE
docker not found.

### DISCUSSION THREAD

**Discussion by AHuusom** [actions/virtual-environments#3958] [Image configuration]
When running a pipeline in azure devops i get this: ##[error]Unhandled: Unable to locate executable file: 'docker'. Please verify either the file path exists or the file can be found within a directory specified by the PATH environment variable. Also check the file mode to verify the file is executable.

When I log in to the ubuntu server and run sudo docker I get command not found.

I have no idea how to fix this, as I don't speak ubuntu very well.

Any help please?

**Comment by miketimofeev** [comment]
@AHuusom could you provide your usage scenario?
